package.json

```diff
@@ -1,6 +1,6 @@
 {
  "name": "voice-desktop-app",
- "version": "1.3.1",
+ "version": "2023.104.20",
```

Comment:
I believe that as long as the leading `0` is there for the month, you don't need the leading 1